bims/static/js/views/detail_dashboard/site_detail.js

```diff
@@ -604,6 +608,137 @@ define([
+            var sub_water_management_areas = siteDetailsWrapper.find(
+                '#sub_water_management_areas');
+            sub_water_management_areas.append(this.renderTableFromTitlesValuesLists(
+                data['site_details']['sub_water_management_areas']));
+            var sa_ecoregions = siteDetailsWrapper.find('#sa-ecoregions');
+            sa_ecoregions.append(this.renderTableFromTitlesValuesLists(
+                data['site_details']['sa_ecoregions']));
+
+            var recordSitesWrapper = $('#fish-ssdd-records-sites');
+            var recordSitesSub = recordSitesWrapper.find('#records-sites')
+            recordSitesSub.append(this.renderTableFromTitlesValuesLists(data['site_details']['records_and_sites']))
+
+            var originsWrapper = $('#fish-ssdd-origins');
+            var originsSub = originsWrapper.find('#origins');
+            originsSub.append(this.renderTableFromTitlesValuesLists(data['site_details']['origins_data'], data,  'origin', false));
+
+            var conservation_statusWrapper = $('#fish-ssdd-conservation-status');
+            var conservation_statusSub = conservation_statusWrapper.find('#ssdd-conservation-status');
+            conservation_statusSub.append(this.renderTableFromTitlesValuesLists(data['site_details']['conservation_status_data'], data, 'cons_status', false));
+
+        },
+
+        renderSiteDetailInfo: function (data) {
+            var $detailWrapper = $('<div></div>');
+            if (data.hasOwnProperty('site_detail_info')) {
+                var siteDetailsTemplate = _.template($('#site-details-template').html());
+                $detailWrapper.append(siteDetailsTemplate({
+                    'fbis_site_code' : data['site_detail_info']['fbis_site_code'],
+                    'site_coordinates' : data['site_detail_info']['site_coordinates'],
+                    'site_description' : data['site_detail_info']['site_description'],
+                    'geomorphological_zone' : data['site_detail_info']['geomorphological_zone'],
+                    'river' : data['site_detail_info']['river'],
+                }));
+            }
+            return $detailWrapper;
+         },
+
+         createDataSummary: function (data) {
+            var bio_data = data['biodiversity_data'];
+            var origin_pie_canvas = document.getElementById('fish-ssdd-origin-pie');
+            this.renderPieChart(bio_data, 'fish', 'origin', origin_pie_canvas);
+
+            var endemism_pie_canvas = document.getElementById('fish-ssdd-endemism-pie');
+            this.renderPieChart(bio_data, 'fish', 'endemism', endemism_pie_canvas);
+
+            var conservation_status_pie_canvas = document.getElementById('fish-ssdd-conservation-status-pie');
+            this.renderPieChart(bio_data, 'fish', 'cons_status', conservation_status_pie_canvas);
+         },
+
+         renderPieChart: function(data, speciesType, chartName, chartCanvas) {
+            if (typeof data == 'undefined') {
```

Comment:
@Christiaanvdm you can clear the canvas before rendering the pie chart like this
```
chartCanvas = this.resetCanvas(chartCanvas);
```